test/lakefsfs/src/main/scala/io/lakefs/fs/LakeFSFS.scala

```diff
@@ -36,11 +37,16 @@ object LakeFSFS {
 
     LOG.info("Create repository={}, branch={}, storageNamespace={}", repository, branch, storageNamespace)
     val repositoriesApi = new RepositoriesApi(apiClient)
-    val repositoryCreation = new RepositoryCreation()
-      .name(repository)
-      .defaultBranch(branch)
-      .storageNamespace(storageNamespace)
-    repositoriesApi.createRepository(repositoryCreation, false)
+    try {
+      val repositoryCreation = new RepositoryCreation()
+        .name(repository)
+        .defaultBranch(branch)
+        .storageNamespace(storageNamespace)
+      repositoriesApi.createRepository(repositoryCreation, false)
+      LOG.info("Repository created repository={}, branch={}, storageNamespace={}", repository, branch, storageNamespace)
+    } catch {
+      case e: ApiException => LOG.error("Create repository failed (code " + e.getCode() + ")", e)
```

Comment:
Please log the actual message, not just the code.

Reply:
The message is part of Java's Exception, passed it as part of the last argument in the log call.